Blink/TermController.swift

```diff
@@ -280,8 +281,24 @@ let _apiRoutes:[String: (MCPSession, String) -> AnyPublisher<String, Never>] = [
   "completion.for": Complete.forAPI
 ]
 
+// MARK: - TermDeviceDelegate methods
+
 extension TermController: TermDeviceDelegate {
   
+  /**
+   When a `ring-bell` notification has been received on `TermView` react to it by sounding a bell if the terminal that sent it
+   is in focus and if it's not send a notification. Tapping the notification opens the session that sent it.
+   */
+  func viewDidReceiveBellRing() {
+    
+    if let isFocused = _proxyView.controlledView?.isFocused, isFocused == false, BKDefaults.notificationOnBellNotInFocus() {
+      viewNotify(["title": "Bell rang", "body": "Tap to open the session"])
+    } else if BKDefaults.playSoundOnBell() {
+      AudioServicesPlaySystemSound(1103);
+      UINotificationFeedbackGenerator().notificationOccurred(.success)
```

Comment:
I think it should be separate options (vibration and sound). And it should be on iPhones only